tg_react/api/accounts/urls.py

```diff
@@ -1,18 +1,28 @@
-from django.conf.urls import url
+from django.conf.urls import re_path
 
-from .views import UserDetails, AuthenticationView, LogoutView, SignUpView, SetLanguageView
+from .views import (
+    UserDetails,
+    AuthenticationView,
+    LogoutView,
+    SignUpView,
+    SetLanguageView,
+)
 from .views import ForgotPassword
 from .views import RestorePassword
 
 
 urlpatterns = [
-    url(r'^me$', UserDetails.as_view(), name='api-user-details'),
-    url(r'^login$', AuthenticationView.as_view(), name='api-user-login'),
-    url(r'^lang$', SetLanguageView.as_view(), name='api-user-language'),
-    url(r'^logout$', LogoutView.as_view(), name='api-user-logout'),
+    re_path(r"^me$", UserDetails.as_view(), name="api-user-details"),
+    re_path(r"^login$", AuthenticationView.as_view(), name="api-user-login"),
+    re_path(r"^lang$", SetLanguageView.as_view(), name="api-user-language"),
+    re_path(r"^logout$", LogoutView.as_view(), name="api-user-logout"),
```

Comment:
Just out of curiosity: why not change them to `path` instead of a bit harder to read `re_path`?

Reply:
I also thought about it. One guess is that `path` would interpret URLs like `/meblah` as `/me`, unless you use `path('/me/')`, which would be backwards incompatible. But at the same time as long as we don't hardcode URLs and rely on urlconf names, this shouldn't be an issue.

Reply:
I chose `re_path` since the semantics of it match `url`. This way can be sure to not introduce weird incompatibilities like the `/meblah` mentioned above.

For example if user has a path in their own urlconf:

```
url(r'^api/user/mewtwo$', MyPokemonView.as_view(), name='api-user-mewtwo'),
```

after upgrading tg-react their view could be "hidden" since it would be matched by `api-user-details`